pandas-stubs/api/indexers/__init__.pyi

```diff
@@ -1,6 +1,6 @@
-from pandas.core.indexers import check_array_indexer as check_array_indexer
-from pandas.core.window.indexers import (
+from pandas.core.indexers import (
```

Comment:
`indexers` is a file in the stubs but a folder in pandas. `BaseIndexer` and so on are technically important from a file in this folder not from `__init__`.